### DISCUSSION TITLE
Folder name for books in series

### DISCUSSION THREAD

**Discussion by Neurrone** [seanap/Plex-Audiobook-Guide#64] [Q&A] [answered]
I notice it currently uses the following naming scheme: `<year> - <title> [<seriesname> <position>]`

This doesn't work well when there are multiple books in a year, as the books will probably not be in the correct order.

Does Booksonic and such have requirements for having the year first?

If not, I am thinking of changing the naming scheme to something like this: `<position> <title> (year)` instead. The series information is already contained in the parent folder name.

**Comment by seanap** [comment, marked as answer]
I've gone back and forth on this as well, there's no issue with your naming scheme.  Booksonic doesn't require the year first structure, here is a quote from the dev: 

> "I like to use the format “year – Bookname”, that way I can quickly see what order they should be listened to directly in my file manager but you can use just the bookname as well if you want. Booksonic will get the year from the tags inside the files."

**Reply by Neurrone** [reply]
Thanks. Do you know if this is also the case for Audiobookshelf? Just trying to make sure that whatever format I end up with is as compatible as possible.

**Reply by seanap** [reply]
Audiobookshelf is more forgiving than booksonic and supports many different folder structures here is a link to their doc's which cover some examples https://www.audiobookshelf.org/docs#naming

![image](https://user-images.githubusercontent.com/17012946/156397740-44e241b1-b33f-4fad-adbd-b8cf4396131f.png)
